dmapiclient/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '25.6.0'
+__version__ = '25.7.0'
```

Comment:
I should have said there are going to be other tickets to update the filters so there is no need to change the version just yet